src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -114,7 +114,7 @@ export default class Pilot {
      * @param airportModel {AirportModel}
```

Comment:
Be s Re to update the doc blocks if you're adding parameters

Reply:
done